doc/classes/Node.xml

```diff
@@ -147,7 +147,7 @@
 			<argument index="2" name="legible_unique_name" type="bool" default="false">
 			</argument>
 			<description>
-				Adds a child node. The child is placed below the given node in the list of children.
+				Adds a [code]child_node[/code]. The child is placed below the given [code]node[/code] in the list of children.
```

Comment:
I think the first part would make more sense as "`Adds [code]child_node[/code] as a child`".